custom/icds/messaging/indicators.py

```diff
@@ -59,10 +59,9 @@ def _get_report_fixture_for_user(domain, report_id, ota_user):
     :param report_id: the index to the result from get_report_configs()
     :param ota_user: the OTARestoreCommCareUser for which to get the report fixture
     """
-    xml = ReportFixturesProviderV1().report_config_to_fixture(
-        get_report_configs(domain)[report_id], ota_user
-    )
-    return etree.tostring(xml)
+    config = get_report_configs(domain)[report_id]
+    fixtures = ReportFixturesProviderV1().report_config_to_fixtures(config, ota_user)
+    return etree.tostring(fixtures[ReportFixturesProviderV1.report_fixture_id(config.uuid)])
```

Comment:
since `ReportFixturesProviderV1` only ever returns a single fixture it would have been easy enough to unpack the list.

Reply:
My thought was that the V1 code won't exist forever, might as well support both versions.

Reply:
Since that requires changing the class here I think this is safe for now. When that change happens it can update the implementation accordingly (eg. `fixture = fixtures[1]`)

Reply:
It seems fragile to depend on the data fixture being second in the list. Could you say more about why you prefer the approach in https://github.com/dimagi/commcare-hq/pull/27375? When I saw your comment "The error is from the xml serialization and not quickcache." earlier I thought this fix was wrong and quickly approved the other PR (had just woken up...), but having read through it again, it seems just the description was wrong, and this PR is correct.

Reply:
purely because it's a much simpler approach. The messaging indicators code already completely dependent on the implementation of the fixture code so it doesn't seem bad to have it know the order of the returned fixtures rather than changing a lot more code for just this one custom use case. Having the test case should also provide an additional level of safety (though more tests could be added since the other ones all mock out the fixture generation).